crt_portal/tms/admin.py

```diff
@@ -17,6 +46,7 @@ def tms_detail_url(self, obj):
         return format_html('<a href="%s">View & update</a>' % (reverse('tms:tms-admin-message', kwargs={"tms_id": obj.tms_id})))
```

Comment:
Is the view for the TMS email not done yet? Here's what I see when I click on this link:

<img width="956" alt="Screen Shot 2022-11-04 at 3 41 31 PM" src="https://user-images.githubusercontent.com/14644234/200061790-86cc8c30-9976-4747-bec0-3a905eef7b60.png">

Reply:
This is a known bug, and outside the scope of this PR.